src/extensions/common/hither/createCalculationPool.ts

```diff
@@ -1,5 +1,8 @@
 export interface CalculationPool {
     requestSlot: () => Promise<{complete: () => void}>
+    pause: () => void
```

Comment:
Ability to pause, resume calculation pool. Didn't end up using this, though. But it does seem to work, so maybe in the future.